invokeai/app/services/model_install/model_install_default.py

```diff
@@ -660,13 +661,96 @@ def _probe(self, model_path: Path, config: Optional[ModelRecordChanges] = None):
         except InvalidModelConfigException:
             return ModelConfigBase.classify(model_path, hash_algo, **fields)
 
+    def _check_for_lora_metadata(self, model_path: Path, info: "AnyModelConfig") -> None:
+        """
+        Check for image files (PNG, JPG, WebP) or JSON metadata files with the same name as the LoRA model.
+        If found, extract relevant metadata and update the model configuration.
+        """
+        from invokeai.backend.model_manager.config import ModelType
```

Comment:
Don't use dynamic imports. Move this to the top of the file. Also, the import should be from `invokeai.backend.model_manager.taxonomy`.